### ISSUE TITLE
Add new Poker scale templates to product

### ISSUE THREAD

**Issue by garethaledavies** [ParabolInc/parabol#8550]
Currently we only have a handful of publicly available templates in Sprint Poker. 

As we did with retros, adding more templates to Sprint Poker makes it more valuable.

Here are a selection of scale templates to add to the product. 

**Modified Fibonacci**
0, 0.5, 1, 2, 3, 5, 8, 13, 20, 40, 100, ?, Pass

**Powers of Two**
0, 2, 4, 8, 16, 32, 64, 128, ?, Pass

**Fruit**
🍒, 🍏, 🥭, 🍍, 🍉, ?, Pass

**Animals**
🐭,🐕,🐖,🐴,🐘

**Cost**
🪙, 💵, 💰, 🏦

**Large, Uncertain, Small**
L, S, ?, Pass

**Priority level**
P0, P1, P2, P3, P4, P5, ?, Pass

**Low, Mid, High scale**
Lo, Mid, Hi, ?, Pass

**Thumb scale**
👍, 👎, ?, Pass

**Yes/No scale**
 Y, N, ?, Pass

**Smiles**
😫, 😞, 😐, 😊, 🤩, ?, Pass

**Stars**
1⭐, 2⭐, 3⭐, 4⭐, 5⭐, ?, Pass

**Traffic Lights**
🟢, 🟡, 🔴, ?, Pass

**Ready/Not Ready**
R, NR, ?, Pass

**Growth Share Matrix**
⭐, ❓, 🐮,  🐶

**SWOT**
S, W, O, T, ?, Pass

In our research, there were other complex prioritization frameworks we could create in Poker. I will save these for a separate issue.

**Comment by Dschoordsch** [comment]
Das this only include the scale templates without any use in a predefined poker template? This would limit it's use to custom templates, wouldn't it?

**Comment by ackernaut** [comment]
Yeah, I’m looking at trying some Poker in disguise to support some new market areas. I’ll probably want to ship both specific templates and their matching scales. Eg. RICE prioritization, MoSCoW prioritization. @Dschoordsch I’ll look at writing the migrations, maybe there’s one where we’ve updated scales in the past?

**Comment by ackernaut** [comment]
For MoSCoW, I’m thinking we have a scale with 4 cards on a single dimension like ‘Priority’: M, S, C, W (must have, should have, could have, won’t have).

For RICE (Reach, Impact, Confidence, Effort) I’m thinking each issue has those 4 dimensions, and then then using a scale of cards, maybe fibonacci 1, 2, 3, 5, 8 or something similar. They can export and run average scores until we build that in someday (_in the future_, _where everything_’_s better_™).

@garethaledavies have you prototyped any of these scales and templates? I’m gonna create a few custom ones as a prototype for RICE and MoSCoW.

**Comment by ackernaut** [comment]
[I’ve got some demo meetings up using custom MoSCoW and RICE templates](https://parabol.slack.com/archives/C05QN33LEAX/p1697127930338519) 🔒

**Comment by Dschoordsch** [comment]
We don't have a new migration which adds the templates. These are still stored in rethink, but we only have 1 type of migrations now.
`yarn pg:migrate create addNewPokerScales`. Then we have 2 files where you can copy from:
* old migration where we added different scales: https://github.com/ParabolInc/parabol/blob/33ed3c1c3924025c18576591b0973f94799d2554/packages/server/database/migrations/20201116132414-modifyDefaultPokerTemplate.ts#L116-L117
* new migration where we modified rethink (to copy the connect stuff etc.): https://github.com/ParabolInc/parabol/blob/33ed3c1c3924025c18576591b0973f94799d2554/packages/server/postgres/migrations/1696443115482_addCustomerFeedbackTemplate.ts#L155-L161

**Comment by ackernaut** [comment]
**Priority? Must/Should/Could/Won’t Have: C**
  [See the discussion](https://action.parabol.co/meet/rTgzJqOZoc/estimate/1) in Sprint Poker #​​12

  *Powered by [Parabol](https://www.parabol.co/resources/sprint-poker-meetings)*

**Comment by jordanh** [comment]
Going to keep this one open. This should be easier now that we've moved to Postgres
